tests/foreman/destructive/test_ldap_authentication.py

```diff
@@ -206,39 +218,25 @@ def test_single_sign_on_ldap_ipa_server(
 
     :expectedresults: After single sign on user should redirected from /extlogin to /hosts page
 
+    :BZ: 1941997
+
     """
-    # register the satellite with IPA for single sign-on and update external auth
-    try:
-        run_command(cmd='subscription-manager repos --enable rhel-7-server-optional-rpms')
-        run_command(cmd='satellite-installer --foreman-ipa-authentication=true', timeout=800000)
-        run_command('satellite-maintain service restart', timeout=300000)
-        if is_open('BZ:1941997'):
-            curl_command = f'curl -k -u : --negotiate {target_sat.url}/users/extlogin'
-        else:
-            curl_command = f'curl -k -u : --negotiate {target_sat.url}/users/extlogin/'
-        result = run_command(curl_command)
-        assert 'redirected' in result
-        assert f'{target_sat.url}/hosts' in result
-        assert 'You are being' in result
-    finally:
-        # resetting the settings to default for external auth
-        run_command(cmd='satellite-installer --foreman-ipa-authentication=false', timeout=800000)
-        run_command('satellite-maintain service restart', timeout=300000)
-        run_command(
-            cmd=f'ipa service-del HTTP/{target_sat.hostname}',
-            hostname=settings.ipa.hostname,
-        )
-        run_command(
-            cmd=f'ipa host-del {target_sat.hostname}',
-            hostname=settings.ipa.hostname,
-        )
+    result = target_sat.execute(f'echo {settings.ipa.password} | kinit {settings.ipa.user}')
+    assert result.status == 0
+    if is_open('BZ:1941997'):
```

Comment:
do we still need this if bz was closed_errata?

Reply:
I think we could get rid of it and use the variant with slash to catch possible regressions.